code/modules/clothing/masks/gasmask.dm

```diff
@@ -73,7 +73,6 @@
 	icon_state = "explorer"
 	item_state_slots = list(slot_r_hand_str = "gas", slot_l_hand_str = "gas")
 	armor = list(melee = 10, bullet = 5, laser = 5,energy = 5, bomb = 0, bio = 50, rad = 0)
-	body_parts_covered = HEAD|FACE|EYES

```

Comment:
This will prevent this mask from protecting them from their eyes burning on phoron.

Reply:
Does it not inherit it from the base gasmask?

Reply:
Ouu. Good point. Wait, why was this defined to begin with then?

Reply:
Probably stolen directly from TG?